src/components/atoms/HookFormInputWrapper/HookFormInputWrapper.tsx

```diff
@@ -55,25 +56,41 @@ export default function HookFormInputWrapper<TValues extends FieldValues>({
   } = useController({ name, ...props });
 
   const child = React.Children.only(children);
+
+  const handleHookFormChange = useCallback((
+    event: ChangeEvent<HTMLInputElement>,
+    value: unknown,
+    shouldUpdate?: boolean,
+  ) => {
+    // for react-select compatibility
+    if (
+      isObject(value) &&
+      "action" in value &&
+      value?.action === "select-option"
+    ) {
+      onChange(event);
+    }
+    if (shouldUpdate) {
```

Comment:
if the above and this condition both are true, onChange will be called twice, which is not desired

Reply:
Sorry and Thanks for noticing this mistake. I forgot to add a return